src/Sign.Cli/appsettings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Logging": {
+        "LogLevel": {
+            "Azure.Core": "Critical",
```

Comment:
Should this not be `Error` instead?

Reply:
@dlemstra, changed to `Error`.

As far as other log levels go, can you create a separate issue and include specific suggestions, if you have any?

Reply:
I will do that